client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/ContactDateCard/InteractionCard/AddressGrid/AddressGrid.tsx

```diff
@@ -18,32 +15,12 @@ const AddressGrid : React.FC<Props> = (props: Props) : JSX.Element => {
                 <Grid item xs={6}>
```

Comment:
Y not xs=12

Reply:
not my change, but there is  an 
```
                <Grid item xs={6}/>
```
at the end, so half the row is empty